README.md

```diff
@@ -1 +1,56 @@
+# Kord
+[![Discord](https://img.shields.io/discord/556525343595298817.svg?color=&label=Kord&logo=discord&style=for-the-badge)](https://discord.gg/6jcx5ev)
+
+This is the GitHub origanization for the Kord project, which aims to provide an idiomatic way to interact with the Discord API with Kotlin
+
+## Projects
+
+Currently we have the following projects
+
+| Name        | Description                                           | Repository                                            |
+| ----------- | ----------------------------------------------------- | ----------------------------------------------------- |
+| kord        | Idiomatic Discord API wrapper                         | [kord](https://github.com/kordlib/kord)               |
+| cache       | Caching library used by Kord                          | [cache](https://github.com/kordlib/cache)             |
+| codegen.kt  | Library we use for generating code                    | [codegen-kt](https://github.com/kordlib/codegen-kt)   |
+| .teamcity   | CI Configuration                                      | [.teamcity](https://github.com/kordlib/.teamcity)     |
+| kordx.emoji | Kotlin emoji library, designed for usage with Discord | [kordx.emoji](https://github.com/kordlib/kordx.emoji) |
+
+## License
+
+Unless stated otherwise in the projects Repository our work is licensed under the [MIT License](https://choosealicense.com/licenses/mit/)
+
+## Code of Conduct
+This code of conduct outlines our expectations for all those who want to participate or interact with the Kord project via GitHub Issues, Discussions, Pull Requests, our Discord server or simmilar
+### How to behave
+The following behaviours are expected and requested of all community members:
+
+- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+- Exercise consideration, respect and empathy in your speech and actions. Remember, we have all been through different stages of learning when adopting technologies.
+- Refrain from demeaning, discriminatory, or harassing behaviour and speech.
+- Disagreements on things are fine, argumentative behaviour or trolling are not.
+
+### How not to behave
+- Do not perform threats of violence or use violent language directed against another person.
+- Do not make jokes of sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory nature, or use language of this nature.
+- Do not post or display sexually explicit or violent material.
+- Do not post or threaten to post other people’s personally identifying information ("doxing").
+- Do not make personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+- Do not engage in sexual attention. This includes, sexualised comments or jokes and sexual advances.
```

Comment:
The comma after `includes` makes this line harder to read and should be removed.